server/handlers/policies_handlers.go

```diff
@@ -0,0 +1,106 @@
+package handlers
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/fs"
+	"net/http"
+	"os"
+	"path"
+	"path/filepath"
+
+	"github.com/gorilla/mux"
+	"github.com/layer5io/meshkit/models/meshmodel"
+	"github.com/layer5io/meshkit/models/meshmodel/core/types"
+	"github.com/layer5io/meshkit/models/meshmodel/core/v1alpha1"
+)
+
+func (h *Handler) GetAllMeshmodelPolicy(rw http.ResponseWriter, r *http.Request) {
+	rw.Header().Add("Content-Type", "application/json")
+	enc := json.NewEncoder(rw)
+	typ := mux.Vars(r)["model"]
+	name := mux.Vars(r)["name"]
+
+	res := h.registryManager.GetEntities(&v1alpha1.PolicyFilter{
+		Kind:      name,
+		ModelName: typ,
+	})
+	unique := make(map[string]bool)
```

Comment:
@harkiratsm Why are you not allowing multiple policies with same model name?

Reply:
At first, I wanted to remove the duplicate policy, so this function gives us unique policy.

Reply:
Now considering your thoughts, I will change this behavior.